### DISCUSSION TITLE
Koniec darmowego (TVN Discovery) Playera

### DISCUSSION THREAD

**Discussion by hawkeye116477** [MajkiIT/polish-ads-filter#18655] [General]
Od jutra Player już nie będzie za darmo, ale reklamy nadal będą w tańszej opcji :smiley: 

Tak więc jakby miał ktoś problem z antyadblockiem, to pewnie będzie problem, bo będzie wymagane płatne konto.

Źródło: https://satkurier.pl/news/200763/player-rezygnuje-z-modelu-avod-koniec-bezplatnych-tresci.html

**Comment by xxcriticxx** [comment]
zajebista sprawa bedzie wiecej na torrentach teraz

**Reply by hawkeye116477** [reply]
@xxcriticxx Pora na zmianę avatara :smile:

<img src="https://user-images.githubusercontent.com/19818572/108103795-b9cdc480-708a-11eb-83fe-514c53447857.png" width="250px" height="250px">

**Reply by xxcriticxx** [reply]
@hawkeye116477 nie lubisz sasiadow?

**Reply by hawkeye116477** [reply]
Po prostu ten bardziej do ciebie pasuje :pirate_flag: ...

**Reply by xxcriticxx** [reply]
@hawkeye116477 moze zmienie na muminki? co myslisz?

**Comment by hawkeye116477** [comment]
Chyba teraz się szykuje podwyżka, albowiem nadchodzi fuzja Warner Media (HBO, Cartoon Network) i Discovery...

**Reply by xxcriticxx** [reply]
HBO MAX

**Reply by hawkeye116477** [reply]
> Od października br. cena pakietu Player bez reklam (bez kanałów TV) wzrosła z 15 zł do 18 zł za 30 dni dostępu. Ceny pozostałych pakietów pozostały na niezmienionym poziomie.

Źródło: https://satkurier.pl/news/209373/player-podniosl-cene-pakietu-bez-reklam.html

**Reply by krystian3w** [reply]
Już jest droższa promocja "udawane pół roku za darmo" z 30 zł podbili na 49 zł...

**Reply by xxcriticxx** [reply]
wiekszosc i tak dostępne na torrentach

**Reply by hawkeye116477** [reply]
@xxcriticxx Nieładnie :pirate_flag: ...

To Amazon teraz jest taniej, 50 zł na rok (+darmowy miesiąc), a są tam gry w formie klucza przeznaczonego do aktywacji na konkretnej platformie (GOG i inne) oraz filmy i darmowa wysyłka.

**Reply by xxcriticxx** [reply]
@hawkeye116477 amazon mam od juz od lat
